wodles/oscap/oscap.py

```diff
@@ -8,16 +8,16 @@
 from re import compile
 from sys import argv, exit, version_info
 from os.path import isfile, isdir
-from tempfile import mkstemp
 from xml.etree import ElementTree
-from os import remove, close as close
-from subprocess import call, CalledProcessError, STDOUT
+from os import remove, close as close, pipe, fork, mkfifo
```

Comment:
Do we actually need `pipe` and `fork`?

Reply:
Not actually. Both are *unused imports* that I didn't see when I was removing dependencies.